test/test_helper.rb

```diff
@@ -1,7 +1,7 @@
 # typed: strong
 # frozen_string_literal: true
 
-require 'simplecov'
+require 'simplecov' unless defined?(Mutant)
```

Comment:
@mbj Can you think of a reason why this wouldn't work?

I'd rather do this than use `$PROGRAM_NAME` because it requires a lot more boilerplate to make sorbet happy.